libs/api/domains/vehicles/src/lib/api-domains-vehicles.service.ts

```diff
@@ -81,8 +82,8 @@ export class VehiclesService {
       const { data } = res
       if (!data) return null
       return data[0]
-    } catch (e) {
-      return this.handle4xx(e, 'Failed to get vehicle search')
+    } catch (error) {
+      return this.handleError(error, 'Failed to get vehicle search')
```

Comment:
HandleError contains a logger.error function, but I don't think this error message, or the error messages below, are adding any useful logging messages that'll assist in debugging that the client fetch error log won't already have provided. I would either like a more descriptive error message for the logs, or just skipping the logging if not possible.